requestmanager/asyncloader/asyncloader_test.go

```diff
@@ -370,7 +384,8 @@ func withLoader(st *store, exec func(ctx context.Context, asyncLoader *AsyncLoad
 	ctx := context.Background()
 	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
 	defer cancel()
-	asyncLoader := New(ctx, st.loader, st.storer)
+	allocator := allocator.NewAllocator(256*(1<<20), 16*(1<<20))
```

Comment:
Even though this is test code, it would be nice to define these with const values